op_crates/web/lib.rs

```diff
@@ -3,18 +3,20 @@
 use deno_core::js_check;
 use deno_core::CoreIsolate;
 use std::path::PathBuf;
+use url::Url;
 
 pub fn init(isolate: &mut CoreIsolate) {
   let files = vec![
     get_path("00_dom_exception.js"),
     get_path("01_event.js"),
     get_path("02_abort_signal.js"),
     get_path("08_text_encoding.js"),
+    get_path("09_location.js"),
   ];
   for file in files {
     println!("cargo:rerun-if-changed={}", file.display());
     js_check(isolate.execute(
-      &file.to_string_lossy(),
+      Url::from_file_path(&file).unwrap().as_str(),
       &std::fs::read_to_string(&file).unwrap(),
     ));
```

Comment:
@bartlomieju There's a bug on Windows where any error with an `op_crates/web` source file in the stack will [panic on source mapping](https://github.com/denoland/deno/runs/1078336836).
```
> deno.exe eval "new Event() // Missing arg."
thread 'main' panicked at 'not implemented: Don't know how to create cache name for scheme: d', cli\disk_cache.rs:107:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```
The specifier you pass here will be parsed as a URL during source mapping, which would cause an invalid scheme error for drive letter paths.

I've fixed that with `Url::from_file_path()` which correctly converts a drive letter file URL.

Reply:
Noticed this in coverage too, op_crates/web are the only files that have non url based specifiers when asking the inspector about things.

Reply:
The change I'm highlighting fixes it.

Reply:
@nayeemrmn nice catch! Could you factor out this change to a separate PR so it could be landed independently? I'm not sure we'll go along with `--location` for 1.4.0, but looks like it fixes actual bug.